core/src/toga/app.py

```diff
@@ -553,6 +553,19 @@ def startup(self):
 
         self.main_window.show()
 
+    def _startup(self):
+        """Since startup can and is overridden by users, verify
+        they did the right thing in their code
+        """
```

Comment:
Since `_startup()` isn't a public method, we lean away from docstrings in favor of inline comments:
```suggestion
# This is a wrapper around the user's startup method that performs any post-setup validation.
```